client/src/presentation/listeners/select-listener.js

```diff
@@ -0,0 +1,6 @@
+import { changeColor } from '../handlers/select-handler.js';
+
+// changing the color of answers div for correct and incorrect answers
+document.querySelectorAll('.div-answer').forEach((element) => {
+  element.addEventListener('click', changeColor);
```

Comment:
after having a look ...I thought that it's better to add class `disable` to all divs, including the clicked one. it just would look more natural

Reply:
Yes you are right.